aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -192,18 +192,63 @@ std::tuple<Tensor, Tensor> slogdet(const Tensor& self) {
   return at::linalg_slogdet(self);
 }
 
-Tensor linalg_pinv(const Tensor& input, const Tensor& rcond, bool hermitian) {
+namespace {
+
+// This function extracts the optional Tensors for atol and rtol
+// Default value for atol is zero
+// Default value for rtol is eps*max(rows, cols)
+// If atol is specified and rtol is not specified then default value for rtol is zero
```

Comment:
I believe this is not documented. Could you add it?

Reply:
Added a sentence "If :attr:`rtol` is not specified and :attr:`atol` is specified to be larger than zero then
:attr:`rtol` is set to zero."